src/navigation/single_fiducial.py

```diff
@@ -12,7 +12,7 @@
 class SingleFiducialStateTransitions(Enum):
     _settings_ = NoAlias
 
-    finished_fiducial = "WaypointState"
+    finished_fiducial = "SinglePostState"
```

Comment:
I think we actually want to switch into SinglePostState from Search if we are looking for a gate. We should only enter single_fiducial if we are just driving to one post so I don't think we want to change its transitions. Let me know if you had a different idea though.

Reply:
Yeah, I think you're right - I'll commit that in

Reply:
Although, I probably didn't do that originally because SearchState goes to SingleFiducial when it finds a tag; are u saying to ideally just go straight to SinglePost instead and have singlefiducial be a more standalone thing?

Reply:
it goes to single fiducial when it finds a tag and its also in post mode. We still want it to do that, but now if its in gate mode we want to transition into this state instead of staying in search as it does currently